RealmModelGenerator.xcodeproj/project.pbxproj

```diff
@@ -605,6 +605,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = macosx;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 3.0;
```

Comment:
Swift 4.0 is the newest version. Lets migrate to that.

Reply:
That's the next pull request not this one.

Reply:
👍